api-mocks/README.md

```diff
@@ -0,0 +1,31 @@
+# API Mocks
+
+CogBoard uses WireMock for testing. As corresponding functional tests are required for acceptance of new widget some of those tests will require mocks of services the widget uses.
+
+## File structure
```

Comment:
* Mocked responses are stored under `__files` directory
* Response files are organized in folders: `widget_name/version`
* Each widget can have multiple `version` directories if it supports many versions of API

Reply:
I'm not fully sure that this order is better in longer perspective. Most day to day apps have multiple services that, because of this I'd like to keep a more of an `app/version/` kind of a style. If there is only one widget for a given app, then sure: `widget_name/version` makes sense.